doxygen/lang/900_release_notes.dox.tmpl

```diff
@@ -301,6 +301,7 @@
       - @ref Qore::ParseOptionStringMap
       - @ref Qore::PO_BROKEN_LIST_PARSING
       - @ref Qore::PO_BROKEN_LOGIC_PRECEDENCE
+      - @ref Qore::PO_BROKEN_LOGIC_PRECEDENCE_WARN
```

Comment:
this needs to be in the new release notes for 0.8.12.11, we shouldn't update release notes for releases already out (for a long time :))